R/LSgameofchance.R

```diff
@@ -104,7 +101,7 @@ compare_function2 <- function(k1, t, p, simulation){
   ## Calculating the probability using negative multinomial distribution and resursive function
   p_calculated <- 0
   for (l in 1:(prod(k)/k[1])){
-    p_calculated <- p_calculated + exp(dnegmn(Y = combinations(k)[l,2:length(k)],
+    p_calculated <- p_calculated + exp(MGLM::dnegmn(Y = combinations(k)[l,2:length(k)],
```

Comment:
for example here, you don't have to load the whole package, just call the function out of it